test_cast/common/framework.py

```diff
@@ -0,0 +1,34 @@
+from . import io
+
+class Framework(object):
+  @staticmethod
+  def launch_eager(input_data, attr):
+    pass
+
+  @staticmethod
+  def launch_static(input_data, attr):
+    pass
+
+class Attribute(object):
+  def __init__():
+    pass
+
+class Runtime(object):
+  def __init__(self, mode, attr, func):
+    self.mode = mode
+    self.attr = attr
+    self.func = func
+  
+  def mode(self):
```

Comment:
Bug here. Invalid definition because of the duplicate member name. But it does not matter in test.